packages/ui/src/Layouts/ColumnLayout.tsx

```diff
@@ -48,6 +48,10 @@ export interface ColumnLayoutProps {
    * Whether to center children over the cross axis, instead of stretching them.
    */
   readonly center?: boolean
+  /**
+   * Set children position horizontally
+   */
+  readonly justify?: 'flex-start' | 'flex-end' | 'space-between' | 'center'
```

Comment:
more appropriate naming would be `align`, when you see `justify` prop on a component, it's not obvious what is the prop for.